### PR TITLE
edit highlights on homepage

### LINKED ISSUE #1784: Revamp highlights on website homepage
It will be nice to revamp the main homepage to highlight features that may be more relevant today compared to 5 years ago.

Current state:

Screenshot

To start with I'm going to document some of the suggestions that have come up. We can have some discussion on this issue and then finalize the 6 highlights we want to have with associated text.

- Online schema changes
- While Shard management sort of gets at this, we want to highlight how easy it is to migrate into Vitess, and to "MoveTables", with close to zero downtime.
- High availability is missing (sort of covered under Manageability by saying "Failovers")
- Backups and restores should be their own highlight
- Connection Pooling can become a feature under performance instead of its own section.
- Workflow can probably be removed
- Materialized Views
- Messaging
- Vindexes aka flexible sharding keys. We say "without adding sharding logic to application", but don't highlight how flexible vitess sharding is

Thanks to @timvaillancourt for raising this and providing many of these suggestions. And to @derekperkins and @shlomi-noach for their follow up suggestions and commentary.

### PR DESCRIPTION
**Pull request by deepthi**
As discussed in #1784, I've made some edits to the highlights section on the homepage.

Fixes #1784

### CODE REVIEW

**Review by frouioui — APPROVED** [review]
Those are good improvements.

**Comment by deepthi** [comment]
Thanks @frouioui!
I'd like to get some more feedback from maintainers before we merge this. So that we don't have to do any follow up PRs.

File: config.toml [hunk 1]
```diff
@@ -65,32 +65,32 @@ language_alternatives = ["en"]
...
 [[languages.en.params.features]]
 title = "Scalability"
-description = "Vitess combines many important MySQL features with the scalability of a NoSQL database. Its built-in sharding features let you grow your database without adding sharding logic to your application."
+description = "Vitess is compatible with MySQL while extending its scalability. Its built-in sharding lets you grow your database without adding sharding logic to your application."
 icon = "cubes"
 
 [[languages.en.params.features]]
 title = "Performance"
-description = "Vitess automatically rewrites queries that hurt database performance. It also uses caching mechanisms to mediate queries and prevent duplicate queries from simultaneously reaching your database. Performance is monitored through nightly <a href='https://benchmark.vitess.io'>benchmarks</a>."
+description = "Vitess improves on MySQL performance through query rewriting, caching and connection pooling. This allows the number of concurrent client connections to scale to orders of magnitude higher than traditional MySQL. Performance is monitored through nightly <a href='https://benchmark.vitess.io'>benchmarks</a>."
```
Comment (mhmohona) [inline config.toml:73]:
```suggestion
description = "Vitess improves MySQL performance through query rewriting, caching and connection pooling. This allows the number of concurrent client connections to scale to orders of magnitude higher than traditional MySQL. Performance is monitored through nightly <a href='https://benchmark.vitess.io'>benchmarks</a>."
```

File: config.toml [hunk 2]
```diff
@@ -65,32 +65,32 @@ language_alternatives = ["en"]
...
-description = "Vitess automatically handles functions like failovers and backups. It uses a topology server to track and administer servers, letting your application be blissfully ignorant of database topology."
+title = "High Availability"
+description = "Vitess automatically detects and repairs primary failures. Data migrations into Vitess, and resharding operations within Vitess are all accomplished with near-zero downtime."
 icon = "tachometer-alt"
 
 [[languages.en.params.features]]
-title = "Connection pooling"
-description = "Vitess eliminates the high-memory overhead of MySQL connections. Vitess servers easily handle thousands of connections at once."
+title = "Features"
+description = "Vitess improves on MySQL by providing materialized views and messaging features that work across shards."
```
Comment (mhmohona) [inline config.toml:83]:
```suggestion
description = "Vitess improves MySQL by providing materialized views and messaging features that work across shards."
```

**Review by mhmohona — COMMENTED** [review]
Hey @deepthi, I just left 2 tiny suggestions. Hope it is helpful. 😄

File: config.toml (hunk 1 above)
Comment (deepthi) [reply config.toml:73]: The way this is written originally is grammatically correct. The meaning changes if you remove the "on".

**Pull request merged**